node.go

```diff
@@ -2,57 +2,67 @@ package main
 
 import (
 	"context"
-	"crypto/rand"
-	"crypto/subtle"
 
 	"github.com/CCI-MOC/obmd/internal/driver"
+	"github.com/CCI-MOC/obmd/token"
 )
 
 // Information about a node
 type Node struct {
 	ConnInfo     []byte             // Connection info for this node's OBM.
 	ObmCancel    context.CancelFunc // stop the OBM
 	OBM          driver.OBM         // OBM for this node.
-	CurrentToken Token              // Token for regular user operations.
+	CurrentToken token.Token        // Token for regular user operations.
 }
 
 // Returns a new node with the given driver information, with no valid token.
```

Comment:
looks like it's now returning a valid token?

Reply:
Fixed.